doc/source/conf.py

```diff
@@ -96,7 +96,7 @@
 
 # Intersphinx mapping
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3", None),
+    "python": ("https://docs.python.org/3.13", None),
```

Comment:
```suggestion
    "python": ("https://docs.python.org/3", None),
```
I would keep it as ``3`` rather than pointing specifically to ``3.13``.